sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -227,6 +230,10 @@ def _test_extrig(f, i, e):
     # because the other expansions undo the unevaluated Mul
     assert expand_trig(f(Mul(i, 1, evaluate=False))) == e
     assert abs(f(i) - e).n() < 1e-10
+    assert cos(x).rewrite(besselj) == sqrt(pi*x/2)*besselj(-S.Half, x)
+    assert cos(x).rewrite(besselj).subs(x, pi/3).n() == S.Half.n()
+    assert cos(sin(x)).rewrite(besselj).subs(x, 2).n() == cos(sin(2)).n()
+    assert cos(cosh(x)).rewrite(besselj).subs(x, 2).n() == cos(cosh(2)).n()
```

Comment:
The subs and .n calls should be removed. The test should be for what the new expression is, not for numerical evaluation.

Reply:
I am running the equation 
```python
sin(cos(x)).rewrite(besselj)
```   

However, I am receiving an output`2**(1/4)*pi**(3/4)*sqrt(sqrt(x)*besselj(-1/2, x))*besselj(1/2, sqrt(2)*sqrt(pi)*sqrt(x)*besselj(-1/2, x)/2)/2`  that is not equal to `sin(cos(x)).rewrite(besselj)` I suspect that `2**(1/4)` is being changed into a numerical value. I have tried using simplify(evaluate = false) but it did not work. Could you please suggest a solution to deal with this issue?